blackbox/docs/release_notes/2.0.0.txt

```diff
@@ -299,3 +302,19 @@ sets both the minimum and maximum memory to allocate, and should be set to
 whatever your previous ``CRATE_MAX_MEM`` was set to.
 
 .. _enterprise edition: https://crate.io/enterprise-edition/
+
+Cluster name in path data
+-------------------------
+
+The computation of the effective data directory path has changed in a way that
```

Comment:
I still think that the phrasing here could be improved a bit. It reads like "the cluster name can no longer be part of the data.path directory, but there is a fallback so it still can be part of it"